source/utilities.cc

```diff
@@ -2149,8 +2149,8 @@ namespace aspect
       time_dependent = false;
       // Give warning if first processor
       this->get_pcout() << std::endl
-                        << "   Loading new data file did not succeed." << std::endl
-                        << "   Assuming constant boundary conditions for rest of model run."
+                        << "   From this timestep onwards, ASPECT will not attempt to load new Ascii data files." << std::endl
```

Comment:
The original message said that reading another file did not succeed. I suspect that this is because there are no more such files. Can you mention that in the error message on *why* you won't attempt to load new files?

Reply:
Good idea. I've added a couple of lines which cover all the present uses of the function.